ui/plugins/eu.esdihumboldt.hale.ui.service.helper/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,12 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Hale UI Service Implementer
+Bundle-SymbolicName: eu.esdihumboldt.hale.ui.service.helper
+Bundle-Version: 3.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-Vendor: wetransform GmbH
+Import-Package: eu.esdihumboldt.hale.common.align.model,
```

Comment:
By convention these projects/bundles should rather be placed in `common` instead of `ui` and named accordingly.